packages/jellyfish-api-core/__tests__/category/icxorderbook/createOrder.test.ts

```diff
@@ -0,0 +1,174 @@
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import {
+  ICXGenericResult, ICXOfferInfo, ICXOrderInfo, ICXOrder, InputUTXO
+} from '../../../src/category/icxorderbook'
+import BigNumber from 'bignumber.js'
+import { accountDFI, checkBTCSellOrderDetails, checkDFISellOrderDetails, idDFI, setup } from './common.test'
+import { RpcApiError } from '../../../src'
+
+describe('Should test ICXOrderBook.createOrder', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup(container)
```

Comment:
Do we need the setup yet? If not remove this and the file and introduce it slowly when it's required.

Reply:
This is like the initial setup. haven't checked what would happen if we remove this. But it just the initial setup in the system that requires to facilitate ICX transactions.